CONTRIBUTING.md

```diff
@@ -1,5 +1,9 @@
-Contribution guidelines
+Guidelines for contributing to webteam-practices
 ===
 
+This repository contains best practices specifically for the Canonical Webteam.
 
+Anyone is welcome to suggest improvements in one of two ways:
 
+- Start a discussion by submitting [a new issue](https://github.com/ubuntudesign/webteam-practices/issues/new)
+- Suggest a change by submitting a pull request ([how?](https://help.github.com/articles/about-pull-requests/))
```

Comment:
perhaps add that 'all PRs that contain substantive changes will be reviewed by at least two people before being approved and merged'